README.md

```diff
@@ -52,7 +52,8 @@ Admin Loggers are an embed style logger that will post realtime, detailed events
 presented in a rich-embed. All triggered events contain an executor, the executor is the member who triggered the event;
 for example, if i deleted your message in Discord, then i would be the executor
 
-for example, if a message is deleted, kicked, banned, or even if a channel was modified or created. It will display who
+for example, if a message is deleted, a member is kicked, banned, or even if a channel was modified or created. It will
+display who
```

Comment:
This contains 3 examples which doesn't flow that well, If we could somehow make this flow a lot better and only have like 2 examples, that would be awesome.

Reply:
Please feel free to add a suggestion, because the limit of my English has been reached here

Reply:
"for example, if a message is deleted, It will display who..." maybe ? keep it a short example ? You've already explained what works with this event so having multiple examples despite having already mentioned what is used is kinda redundant.

Reply:
> "for example, if a message is deleted, It will display who..." maybe ? keep it a short example ? You've already explained what works with this event so having multiple examples despite having already mentioned what is used is kinda redundant.

I think i will keep this, and simply it when i have screenshot examples.

Reply:
okii dokii